### PR TITLE
build(docs-infra): implement the 'package' API template

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

**Comment by petebacondarwin** [comment]
See https://pr24631-1d7618a.ngbuilds.io/api/animations

**Comment by petebacondarwin** [comment]
screen shot 2018-06-22 at 18 13 33

File: aio/tools/transforms/angular-api-package/processors/processPackages.spec.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    expect(newDocs[0].pipes).toEqual([
+      { docType: 'pipe', id: 'pipe-1' },
+    ]);
+    expect(newDocs[0].types).toEqual([
+      { docType: 'type-alias', id: 'type-alias-1' },
+      { docType: 'const', id: 'const-1' },
+      { docType: 'const', id: 'const-2' },
+    ]);
+  });
+});
```
Comment (gkalpak) [inline aio/tools/transforms/angular-api-package/processors/processPackages.spec.js]: Missing some primary/secondary tests :grin:

**Review by gkalpak — CHANGES_REQUESTED** [review]
Mostly LGTM, but the CI failures seem legit.

File: aio/tools/transforms/angular-api-package/processors/processPackages.spec.js (hunk 1 above)
Comment (petebacondarwin) [reply aio/tools/transforms/angular-api-package/processors/processPackages.spec.js]: added

**Comment by petebacondarwin** [comment]
You can now, also, filter by package: https://pr24631-e0c72c9.ngbuilds.io/api?query=ani&type=package

screen shot 2018-07-01 at 21 37 30

**Review by gkalpak — APPROVED** [review]
LGTM as soon as Travis is happy.

**Review by matsko — CHANGES_REQUESTED** [review]

**Review by matsko — APPROVED** [review]

**Pull request closed without merging**